app/src/main/java/ch/epfl/sweng/swenggolf/offer/Category.java

```diff
@@ -11,4 +11,18 @@
     public static Category getDefault() {
         return Category.OTHER;
     }
+
+    private int value;
+
+    static {
```

Comment:
Why are you reassigning values to the enum ?

Reply:
Resolved thanks to the forgotten `ordinal()` method.